src/util/Constants.js

```diff
@@ -74,13 +74,13 @@ exports.DefaultOptions = {
    * HTTP options
    * @typedef {Object} HTTPOptions
    * @property {number} [version=7] API version to use
-   * @property {string} [api='https://discordapp.com/api'] Base url of the API
-   * @property {string} [cdn='https://cdn.discordapp.com'] Base url of the CDN
+   * @property {string} [api='https://discord.com/api'] Base url of the API
+   * @property {string} [cdn='https://cdn.discord.com'] Base url of the CDN
    * @property {string} [invite='https://discord.gg'] Base url of invites
    */
   http: {
     version: 7,
-    api: 'https://discordapp.com/api',
+    api: 'https://discord.com/api',
     cdn: 'https://cdn.discordapp.com',
```

Comment:
```suggestion
    cdn: 'https://cdn.discord.com',
```

Reply:
The CDN doesn't move, and if I saw the conversation in DDevs right, it may not move. Remains to be seen

Reply:
CDN isn’t moving, see my comment.